executor/test.h

```diff
@@ -5,6 +5,11 @@
 #include "test_linux.h"
 #endif
 
+#if GOOS_windows
+#define le16toh(x) (x)
+#define htole16(x) (x)
```

Comment:
This file is included into executor.cc and executor.cc already have a similar portability layer for similar macros, please  keep them in 1 place.